input/definitions/MeasureReport/StructureDefinition-measureReport-populationDescription.json

```diff
@@ -0,0 +1,89 @@
+{
+    "resourceType" : "StructureDefinition",
+    "id" : "measurereport-populationDescription",
+    "extension" : [{
+        "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm",
+        "valueInteger" : 3
+      },
+      {
+        "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+        "valueCode" : "cqi"
+      },
+      {
+        "url" : "http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status",
+        "valueCode" : "trial-use"
+      }],
+        "url" : "http://hl7.org/fhir/StructureDefinition/measurereport-populationDescription",
+    "name" : "measurereport-populationDescription",
+    "title" : "Measure Report Population Description",
+    "status" : "active",
+    "experimental" : false,
+    "date" : "2024-02-01",
+    "publisher" : "HL7 International / Clinical Quality Information WG",
+    "contact" : [{
+      "telecom" : [{
+        "system" : "url",
+        "value" : "http://www.hl7.org/Special/committees/cqi"
+      }]
+    }],
+    "description" : "The human readable description of population criteria.",
+    "jurisdiction" : [{
```

Comment:
I wouldn't limit the jurisdiction to the US, this is just a general extension for use in describing population criteria in a measure report

Reply:
Latest commit: 6065105
Used the  http://unstats.un.org/unsd/methods/m49/m49.htm jurisdiction, 001, "World" as used in other extensions